tests/cpp/test_pointwise.cpp

```diff
@@ -1435,4 +1437,200 @@ TEST_F(
   }
 }
 
+using Tma2dTileTestParams =
+    std::tuple<int64_t, int64_t, bool>; // <dim0, ndims, use_tma_store>
+
+class Tma2dTileTest : public NVFuserFixtureParamTest<Tma2dTileTestParams> {
+ protected:
+  void SetUp() override {
+    NVFuserFixtureParamTest<Tma2dTileTestParams>::SetUp();
+    NVFUSER_TEST_CUDA_ARCH_GUARD(9, 0);
+    enable_options_guard_ = std::make_unique<EnableOptionsGuard>();
+    EnableOptionsGuard::getCurOptions().set(EnableOption::TmaPointwise);
+  }
+
+ private:
+  std::unique_ptr<EnableOptionsGuard> enable_options_guard_;
+};
+TEST_P(Tma2dTileTest, NoBroadcast) {
```

Comment:
Can you please add some high-level comment here? For example, what are you trying to test or demonstrate? What patterns are you going to look at and why?